src/bootstrap.php

```diff
@@ -698,6 +698,9 @@ function (Request $request, string $id) use ($app) {
         array_map($addType('interview'), $app['interviews']),
         array_map($addType('labs-experiment'), $app['experiments'])
     );
+    usort($items, function ($a, $b) {
+        return $a['published'] >= $b['published'] ? -1 : 1;
```

Comment:
Easier to use the spaceship operator (`<=>`, like https://github.com/elifesciences/api-dummy/blob/develop/src/bootstrap.php#L92-L95 - maybe comparing strings is enough though).